lib/util.dart

```diff
@@ -152,14 +152,14 @@ class Util {
 
   /// Returns a regular Flutter widget representing this animation, rendered with the specified size.
   ///
-  /// This actually creates an [EmbeddedGameWidget] with a [SimpleGame] whose only content is an [AnimationComponent] created from the provided [animation].
+  /// This actually creates an [EmbeddedGameWidget] with a [SimpleGame] whose only content is an [SpriteAnimationComponent] created from the provided [animation].
   /// You can use this implementation as base to easily create your own widgets based on more complex games.
   /// This is intended to be used by non-game apps that want to add a sprite sheet animation.
   ///
-  @Deprecated('Use SpriteAnimation instead')
-  widgets.Widget animationAsWidget(Position size, Animation animation) {
+  @Deprecated('Use SpriteAnimationWidget instead')
```

Comment:
Since this PR is targeting v1 already, maybe just remove this deprecated method?